pkg/utils/lighthouseutil.go

```diff
@@ -0,0 +1,16 @@
+package utils
+
+import (
+	"github.com/submariner-io/submariner-operator/pkg/subctl/operator/common/embeddedyamls"
+	apiextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+)
+
+func GetMcsCRD() (*apiextensionsv1beta1.CustomResourceDefinition, error) {
+	crd := &apiextensionsv1beta1.CustomResourceDefinition{}
+
+	if err := embeddedyamls.GetObject(embeddedyamls.Lighthouse_crds_multiclusterservices_crd_yaml, crd); err != nil {
+		return nil, err
+	}
+
+	return crd, nil
+}
```

Comment:
I don't want to nitpick but I'd prefer this to be in _pkg/lighthouse/crds.go_ rather than under a general utils package.

Reply:
Done.